src/types/index.ts

```diff
@@ -120,4 +121,5 @@ export type ImportOpenApi = {
   input?: InputOptions;
   output?: OutputOptions;
   workspace: string;
+  converterOptions: any;
```

Comment:
Could be cool if you can add a type for that in the swagger2openapi.d.ts file

Reply:
swagger2openapi.d.ts options parameter is not typed. 
We can create typings like https://github.com/Mermade/oas-kit/tree/master/packages/swagger2openapi#usage But I don't think it's worth it. Because converterOptions used for only one time parameter passing. I think there is nothing code reusability this time or in the future.

Reply:
Ok then maybe just add a comment with a link to the possible options

Reply:
I have added ConverterOptions type on latest commit

Reply:
Awesome!